charts/loki/templates/service-memberlist.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.config.memberlist -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "loki.fullname" . }}-memberlist
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "loki.labels" . | nindent 4 }}
+spec:
```

Comment:
do we want to include `publishNotReadyAddresses` here, to fix the chicken and egg issue when the ring is first being populated? or maybe we don't care?

Reply:
@trevorwhitney I had to do it manually for the query-frontend service to avoid the error below because the chart does not give the chance to add annotations or the publishNotReadyAddresses parameter.
```
level=error ts=2022-05-28T01:05:41.798975654Z caller=frontend_processor.go:69 msg="error processing requests" address=10.20.3.238:9095 err="rpc error: code = Canceled desc = context canceled"
```

Reply:
yes, it's a great idea... added